src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -3878,8 +3878,9 @@ public static int Main()
     }
 }
 ";
-
-            ParseAndValidate(test, Diagnostic(ErrorCode.ERR_NoModifiersOnAccessor, "private"), Diagnostic(ErrorCode.ERR_NoModifiersOnAccessor, "public"));
+            CreateCompilationWithMscorlib(test).GetDeclarationDiagnostics().Verify(
+                Diagnostic(ErrorCode.ERR_NoModifiersOnAccessor, "private"), 
```

Comment:
>Diagnostic(ErrorCode.ERR_NoModifiersOnAccessor, "private") [](start = 16, length = 58)

Please include comments with error message and the actual line of code for the error.